server/utils/initdb.js

```diff
@@ -1,5 +1,8 @@
 import db from '../models';
+import envConfig from '../config/env-config';
 
-db.sequelize.sync().then(() => {
+const params = envConfig.NODE_ENV === 'test' ? { force: true } : {};
```

Comment:
**[comment]** I don't expect us to keep syncing tables in production.

Reply:
We are working on this. This will be fixed in another PR.